api/job/janitor_test.go

```diff
@@ -3,17 +3,48 @@ package job
 import (
 	"testing"
 	"time"
+
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestJanitor(t *testing.T) {
 	store := NewMemoryStore()
 	janitor := NewJanitor(store, time.Hour*24)
 
-	// todo: put some jobs in the store
+	expiry := -24 * time.Hour
+	now := time.Now()
+
+	jobs := []*models.Job{
+		{
+			JobID:   "delete",
+			Created: now.Add(expiry),
+		},
+		{
+			JobID:   "keep",
+			Created: now.Add(expiry * -2),
+		},
+	}
+
+	for _, job := range jobs {
+		store.jobs = append(store.jobs, job)
+	}
 
 	if err := janitor.Run(); err != nil {
 		t.Fatal(err)
 	}
 
-	// todo: assert only old jobs got deleted
+	expected := []*models.Job{
```

Comment:
using the mock store would allow this part to be removed

Reply:
I get an import cycle error when I try to use mock_job.MockJobStore. I wasn't sure how to implement it.

Reply:
ok, then this should just be `assert.Equal(t, expected, tagStore.Tags())`